google-cloud-datastore/src/main/java/com/google/cloud/datastore/DatastoreImpl.java

```diff
@@ -546,6 +593,8 @@ ByteString requestTransactionId(
 
   com.google.datastore.v1.BeginTransactionResponse beginTransaction(
       final com.google.datastore.v1.BeginTransactionRequest requestPb) {
+    Span span = traceUtil.startSpan(TraceUtil.SPAN_NAME_BEGINTRANSACTION);
+    Scope scope = traceUtil.getTracer().withSpan(span);
```

Comment:
Personally, I prefer to use try-with-resources for scope. It implements AutoClosable, so it'll be closed when the try block ends. Ex:
```java
try (Scope s = tracer.withSpan(span)) {
 ... rest of the logic
}
```

Reply:
Done